scripts/ci/run-benchmark

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+"""Script to benchmark several high level cli commands.
+
+As of now this benchmarks `cp` and `rm` with test cases for multiple 4kb files
+(default 10000 files) and a single large file (default 10gb, `cp` only).
+"""
+import os
+from subprocess import check_call
+from datetime import datetime
+import random
+import argparse
+
+TEST_BUCKET = os.environ.get('PERF_TEST_BUCKET')
+MANY_FILES_DIR = "files/10k"
+LARGE_FILE_DIR = "files/10gb"
```

Comment:
Worth using `os.path.join` so these will need less changes when we get these running on windows.  Though you'll need to account for this when creating the `S3` paths.